assets/stylesheets/layout/_main.scss

```diff
@@ -375,10 +376,56 @@ $sidebar-width-min: 228px;
 	}
 }
 
+.focus-design {
+	.wp-primary {
+		opacity: 0.2;
+		pointer-events: none;
+	}
+
+	.wp-secondary .design-menu {
+		opacity: 1;
+		transform: translateX( 272px );
+		pointer-events: auto;
+	}
+
+	.sidebar,
+	.wp-secondary .site-selector {
+		pointer-events: none;
+	}
+}
+
 .focus-sidebar {
 	overflow: hidden;
 }
 
+.design-menu {
+	background: lighten( $gray, 30% );
+	position: fixed;
+		top: 0;
+		bottom: 0;
+		left: -272px;
+	width: 272px;
+	overflow: hidden;
+	opacity: 0.8;
+	z-index: 181;
```

Comment:
Please add new z-indexes to our mapping. https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/css.md#csssass-coding-guidelines

You may find this extension helpful when finding the correct stacking context: https://github.com/gwwar/z-context